pkgs/development/libraries/libpam-wrapper/default.nix

```
@@ -0,0 +1,35 @@
+{ stdenv
+, fetchFromGitLab
+, cmake
+, openpam
+, python3
+}:
+
+stdenv.mkDerivation rec {
+  version = "1.1.3";
+  pname = "libpam-wrapper";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.com";
+    owner = "cwrap";
+    repo = "pam_wrapper";
+    rev = "ab2d90c6def33888db86755e23f3c12d417e187f";
```

Using tags is easier to update:

```suggestion
    rev = "pam_wrapper-${version}";
```